virtual/notification-daemon/notification-daemon-0.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2022 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
```

Comment:
This requires a new revision for the RDEPEND change.

Reply:
Done, thank you!